src/main/scala/cromwell/engine/io/gcs/GcsFileAttributes.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell.engine.io.gcs
+
+import java.nio.file.Path
+import java.nio.file.attribute.{FileTime, BasicFileAttributes}
+
+class GcsFileAttributes(path: Path) extends BasicFileAttributes {
+
```

Comment:
Superfluous newlines throughout. Also if we're going to require a NotImplementedError instead of ??? personally I'd provide some meaningful message there (beyond "foo is not implemented")